iamvane/nabi_ui
# [NABI-16] edit name

Implements [[NABI-16]](https://iamvane.backlog.com/view/NABI-16)
**Main Tasks**
- On hover name, add styles and edit icon
- Add EditName modal
- Add form fiels
- OnClick functionality: dialog opens
- Add styles
- Add/edit unit tests

**To test manually**
Go to `/profile-builder/x` -> click on "Edit" icon next to name -> EditName modal should appear
